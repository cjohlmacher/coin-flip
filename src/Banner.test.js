import { render } from '@testing-library/react';
import Banner from './Banner';

it('renders without crashing', () => {
    render(<Banner />);
});

it('matches snapshot', () => {
    const {asFragment} = render(<Banner title="Coin Flip"/>);
    expect(asFragment()).toMatchSnapshot();
})