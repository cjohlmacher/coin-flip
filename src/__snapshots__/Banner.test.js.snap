// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Banner"
  >
    <h1>
      Coin Flip
    </h1>
  </div>
</DocumentFragment>
`;
